docs/AboutUs.adoc

```diff
@@ -3,8 +3,7 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+AddressBook - Level 4 was developed by the https://github.com/CS2103JAN2018-T09-B3/main[T09-B3] team. +
```

Comment:
We need to write our project reflection instead of Addressbook lvl 4. Just one line but we need to add